view/src/main/webapp/WEB-INF/tags/formatDateTime.tagx

```diff
@@ -10,6 +10,8 @@
     <jsp:useBean id="date" class="java.util.Date"/>
     <c:set target="${date}" property="time" value="${millis}"/>
 
+    <!-- TODO: need to check if ${date} is zero. COLAB-2495 Creation date could be Null. -->
```

Comment:
For new TODOs, we always add an issue code. If there's no issue about this yet, you can add it on Jira and then change it to `TODO COLAB-XXXX:`.

Is this still required though? The creation date should not be allowed to be null, right?

Reply:
Exactly, this was a note to myself earlier on. It's a design decision I guess. The advantage of not doing the check here is that we are more likely to find any issues with the database. :-)

Reply:
I agree, let's keep it this way and not add the check - this way we'll fail early if something is wrong in the DB.